src/Augurk.CommandLine/Commands/PublishCommand.cs

```diff
@@ -252,6 +252,7 @@ private Feature ParseFeatureFile(string featureFile)
         {
             using (var reader = new StreamReader(featureFile))
             {
+                Stack<string> directories = new Stack<string>();
```

Comment:
Check. I created the stack because I anticipated more directory-changes. Will change this.